### PR TITLE
[AWN-43405] Install support libraries into AWN specific path

### PR DESCRIPTION
**Pull request by sean-rootsecure**

### CODE REVIEW

File: RPM/libgcrypt-devel/rpmbuild/SPECS/libgcrypt-devel.spec [hunk 1]
```diff
@@ -19,48 +21,52 @@ in GNU Privacy Guard.  This package contains files needed to develop
...
+mkdir -p %{buildroot}/opt/awn/share/man/man1
+mkdir -p %{buildroot}/etc/ld.so.conf.d
+cp /bin/dumpsexp %{buildroot}/opt/awn/bin/
+cp /bin/hmac256 %{buildroot}/opt/awn/bin/
+cp /bin/libgcrypt-config %{buildroot}/opt/awn/bin/
+cp /bin/mpicalc %{buildroot}/opt/awn/bin/
+cp /usr/include/gcrypt.h %{buildroot}/opt/awn/usr/include/
+cp /lib64/libgcrypt.la %{buildroot}/opt/awn/lib64/
+ln -s /lib64/libgcrypt.so.20.2.6 %{buildroot}/opt/awn/lib64/libgcrypt.so
+ln -s /lib64/libgcrypt.so.20.2.6 %{buildroot}/opt/awn/lib64/libgcrypt.so.20
```
Comment (aschryver) [inline RPM/libgcrypt-devel/rpmbuild/SPECS/libgcrypt-devel.spec]: For completeness, I am going to leave a comment here. Just want to make sure we know the result of changing the links.

Comment (sean-rootsecure) [reply RPM/libgcrypt-devel/rpmbuild/SPECS/libgcrypt-devel.spec:39]: This is actually a bug. Well caught. The libgcrypt.so.20.2.6 file should be located in /opt/awn/lib64, not /lib64

Comment (sean-rootsecure) [reply RPM/libgcrypt-devel/rpmbuild/SPECS/libgcrypt-devel.spec]: You are correct. This is actually a bug:
```# ls -l /opt/awn/lib64/libgcr*
-rwxr-xr-x. 1 root root     954 Sep 23 16:56 /opt/awn/lib64/libgcrypt.la
lrwxrwxrwx. 1 root root      26 Sep 23 18:04 /opt/awn/lib64/libgcrypt.so -> /lib64/libgcrypt.so.20.2.6
lrwxrwxrwx. 1 root root      19 Sep 23 18:25 /opt/awn/lib64/libgcrypt.so.20 -> libgcrypt.so.20.2.6
-rwxr-xr-x. 1 root root 4586416 Sep 23 16:56 /opt/awn/lib64/libgcrypt.so.20.2.6```

**Review by aschryver — APPROVED** [review]
Links are good.

**Pull request merged**